nexus-sdk/src/main/java/io/nexusrpc/OperationUnsuccessfulException.java

```diff
@@ -3,27 +3,21 @@
 /** An operation has failed or was cancelled. */
 public class OperationUnsuccessfulException extends Exception {
```

Comment:
I'd consider renaming this to `OperationException`, In Go I plan to rename to `OperationError` (I'll likely keep an deprecated alias for a bit for `UnsuccessfulOperationError` to prevent the breaking change.

Reply:
OK if I rename `OperationUnsuccessfulException` I would also rename `OperationHandlerException`, @cretz any objection?

Reply:
My concern with these renamings is that they assume a generic-ness about the exceptions. When I'm using a client, is an operation timeout an operation exception? Is a handler exception every used outside of an operation implementation?

Reply:
I think that would be a handler error @bergundy ?

Reply:
I am not a fan of the generic names here myself. That only some operation exceptions become operation exceptions, or that operation handler exceptions are now named as if they apply to all handlers I think is a bit off. Regardless, I am going to mark approved because it was done in Go.